src/operator/src/bulk_insert.rs

```diff
@@ -111,7 +98,8 @@ impl Inserter {
                     body: Some(bulk_insert_request::Body::ArrowIpc(ArrowIpc {
                         region_id: region_id.as_u64(),
                         schema: schema_bytes,
-                        payload,
+                        payload: data.data_body,
+                        data_header: data.data_header,
```

Comment:
```suggestion
                        data_header: data.data_header,
                        payload: data.data_body,
```

This way looks more comfortable.

Reply:
fixed in fb079fe2e